elements/pfe-sass/maps/_general.scss

```diff
@@ -1,26 +1,48 @@
+@import "@patternfly/patternfly/sass-utilities/scss-variables";
+
 $pfe-vars: (
-  container-spacer:              16px,
-  container-padding:             16px,
-  content-spacer:                24px,
 
-  font-size:                     16px,
-  line-height:                   1.5,
+  container-spacer:              pf-font-prem(16px),
+  container-padding:             pf-font-prem(16px),
+  content-spacer:                pf-font-prem(24px),
+  content-spacer--heading--lg:   pf-font-prem(32px),      
+  content-spacer--heading--md:   pf-font-prem(24px),
+  content-spacer--heading--sm:   pf-font-prem(16px),
+
+  font-family: '"Red Hat Text", "RedHatText", "Overpass", Overpass, Arial, sans-serif',
+  font-family--heading: '"Red Hat Display", "RedHatDisplay", "Overpass", Overpass, Arial, sans-serif',
+  font-family--code: '"Overpass Mono", Consolas, Monaco, "Andale Mono", monospace',
+
+  line-height--heading:     1.2,
+  line-height:              1.3,
 
-  font-family:                   '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--heading:          '"Overpass", Overpass, Helvetica, helvetica, arial, sans-serif',
-  font-family--code:             '"Overpass Mono", Consolas, Monaco, 'Andale Mono', monospace',
+  // General font size variables, used with math to set t-shirt variable values
+  root-font-size:           1em,  // must be EM or PX unit, everything else is rems
 
-  font-size--heading--alpha:     32px,
-  font-size--heading--beta:      24px,
-  font-size--heading--gamma:     21px,
-  font-size--heading--delta:     18px,
-  font-size--heading--epsilon:   16px,
-  font-size--heading--zeta:      14px,
+  base--text:               1rem,      // 16px
+  base--heading:            1.25rem,   // 20px
+  base--heading-lg:         1.5625rem, // 25px
```

Comment:
```suggestion
  base--heading--lg:         1.5625rem, // 25px
```

Reply:
hmm, this is the naming convention the redhat-theme is already using.. not sure how to still support that and change this. I can dig into this further to see how complex it is

Reply:
on second thought maybe this is ok. The base variables don't have to match, only the font size ones. Like in the theme, we have `  --pfe-theme--font-size--xxl` which has the value of `calc(1.92  * var(--pfe-theme--base--heading-lg, 25px))`.   It could have the value of `calc(1.92  * var(--unicorn))` and it would still work.

Reply:
I take it back, it does matter. When we apply the theme, the base--text and base--heading sneak in there and have some influence, the only one that stops working is base--heading-lg, which means things get out of whack. I reverted these back.